app/docs/introduction/featured-posts.tsx

```diff
@@ -8,7 +8,7 @@ export async function getFeaturedPosts() {
   const db = client.db("ShadowrunWeb");
   const posts = await db
     .collection("Posts")
-    .find({ isPublished: "yes" })
+    .find({ Published: true })
```

Comment:
I actually like your isPublished column name better! if you want to change it to published, lowercase it too. Follow the camelCase convection with these

Reply:
must have missed that. Will update on next commit